gp/cartesian_graph.py

```diff
@@ -293,7 +303,9 @@ def _f(x):
 
         return locals()["_f"]
 
-    def to_torch(self, parameter_names_to_values=None):
+    def to_torch(
```

Comment:
does this return value annotation cause any problems if torch is not available?

Reply:
probably yes, we currently don't test this in travis, but we should.

Reply:
```python
if torch_available:
    def to_torch(...):
         # full definition
else:
    def to_torch(*args, **kwargs):
        raise NotImplementedError(...)
```

Reply:
But I will investigate if this throws an error.